include/libtorrent/alert_types.hpp

```diff
@@ -271,7 +278,7 @@ namespace libtorrent
 		file_completed_alert(aux::stack_allocator& alloc, torrent_handle const& h
 			, int idx);
 
-		TORRENT_DEFINE_ALERT(file_completed_alert, 6)
```

Comment:
if you have a torrent with lots of files, this could be just as spammy as the per-piece alerts. scaling wise I think bumping this has a weaker justification than the state change alerts (because those scale per-torrent, and won't really happen back-to-back). do you have a good rationale or justification?